modules/programs/zsh.nix

```diff
@@ -511,15 +511,15 @@ in
 
     (mkIf (cfg.dotDir != null) {
       home.file."${relToDotDir ".zshenv"}".text = ''
-        export ZDOTDIR=${zdotdir}
+        export ZDOTDIR="${zdotdir}"
```

Comment:
`escapeShellArg` rather than quotes? Because if the path contains double quote (`"`) this breaks (unlikely, but might as well try to be _more_ correct).

Reply:
Ah good point. I had to call `escapeShellArg` on `cfg.dotDir` directly though instead of on the entire `zdotdir` since we don't want to escape `$HOME`.